toptal/picasso
# feat(ModalContent): add scrollable shades

[SPT-1004](https://toptal-core.atlassian.net/browse/SPT-1004)

### Description

Add vertical shades to scrollable modal content

PS. Manually tests on Chrome, Firefox, and Safari

### How to test

1. Go to the modal example in the storybook Temploy
2. Scroll all the way down to the last modal example
3. Open one of these modals

<img width="531" alt="Screen Shot 2020-10-13 at 23 35 30" src="https://user-images.githubusercontent.com/830217/95936795-c96f9080-0dac-11eb-9535-57bf2d93a27f.png">

4. Make your scroll vertically smaller until the modal content gets scrollable
5. Ensure you see vertical shades that indicate the content is scrollable

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="676" alt="Screen Shot 2020-10-13 at 23 31 59" src="https://user-images.githubusercontent.com/830217/95936577-4c441b80-0dac-11eb-9511-4ca8264097c5.png"> | <img width="700" alt="Screen Shot 2020-10-13 at 23 31 14" src="https://user-images.githubusercontent.com/830217/95936539-32a2d400-0dac-11eb-9617-a4079a764808.png"> |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
